src/test/java/learnositysdk/request/InitTest.java

```diff
@@ -451,6 +452,30 @@ public void testJsonSlashEscaping()
         );
     }
 
+	@Test
+	public void testJsonEmdashEscaping()
+			throws java.lang.Exception
+	{
+		System.out.println("Init: Test that em dashes are not escaped");
+
+		String request = "{\"emdash\":\"–\",\"slash\":\"/\",\"slash-back\":\"\\\\\"}";
+
+		securityObj.put("domain","x");
+
+		init = new Init("items", securityObj, consumerSecret, request);
+
+		String signedRequest = init.generate();
+
+		JSONObject signedReqObject = new JSONObject(signedRequest);
+
+		assertSecurityPacket(signedReqObject.getJSONObject("security").toString());
+
+		assertEquals(expectedEmdashSignature,
+				signedReqObject.getJSONObject("security").getString("signature"),
+				"Invalid signature generated"
+		);
```

Comment:
A few stylistic tweaks to this test
- please reduce the number of blank lines in this function
- the character you're using in this test is actually an `en dash` not an `em dash`
- we tend to use a real domain name in the other tests in this file, `demos.learnosity.com`
- please declare the `expectedEmdashSignature` variable in the scope of this function since it isn't used in multiple functions. This way we can keep the test data closer to the test code.

Reply:
Hi @flakeparadigm, this has been fixed.